src/BsRecharts__Brush.re

```diff
@@ -0,0 +1,79 @@
+/* http://recharts.org/en-US/api/Brush */
+open BsRecharts__Utils
+
+[@bs.module "recharts"] external reactClass : ReasonReact.reactClass = "Brush";
+
+[@bs.obj]
+external makeProps :
+  (
+    ~className: string=?,
+    ~data: array('data)=?,
+    ~dataKey: string=?,
+    ~endIndex: int=?,
+    ~fill: string=?,
+    ~gap: int=?,
+    ~height: int=?,
+    ~onChange: ({..
+      "startIndex": int,
+      "endIndex": int
+    } => unit)=?,
+    ~padding: padding=?,
+    ~startIndex: int=?,
+    ~stroke: string=?,
+    ~tickFormatter: 'tickFormatter=?,
+    ~travellerWidth: int=?,
+    ~updateId: 'updateId /*[@bs.unwrap] [ | `string(string) | `int(int)]*/=?,
```

Comment:
I don't see this in docs, maybe you can give me more info on this one?
can we make it `string`, and if needed consumer will convert `int` to `string` himself, will this work?

Reply:
We can honestly just remove updateId altogether. I got it reading the code directly instead of viewing the API docs. I will remove it and commit.